tests/envdir-url.t

```diff
@@ -24,7 +24,7 @@ Files are populated from NEXTSTRAIN_ENVD_URL.
 
   $ aws s3 cp --quiet env.d.zip "$NEXTSTRAIN_ENVD_URL"
 
-  $ docker run --rm -e NEXTSTRAIN_ENVD_URL --env=AWS_{ACCESS_KEY_ID,SECRET_ACCESS_KEY,SESSION_TOKEN} "$IMAGE" \
+  $ docker run --rm -e NEXTSTRAIN_ENVD_URL --env=AWS_{ACCESS_KEY_ID,SECRET_ACCESS_KEY,SESSION_TOKEN,DEFAULT_REGION} "$IMAGE" \
```

Comment:
> AWS CLI v2 runs into errors if the region is not defined¹

Uh... so does this mean that all our usages of the Docker/AWS Batch/Singularity runtimes which call `aws` are going to have to ensure they have `AWS_DEFAULT_REGION` set? That seems like a unexpected breaking change. Hmm.

Reply:
If that's an issue we don't want to deal with now, we can also [install AWS CLI v1 using the bundled installer](https://docs.aws.amazon.com/cli/v1/userguide/install-linux.html#install-linux-bundled-no-sudo)

Reply:
Maybe? Or maybe we could provide a default `AWS_DEFAULT_REGION` that's backwards compat with v1? (What does it do if region isn't required?) Or maybe we bite the bullet and fix anything that breaks (although this means breaking other people's uses too).

Maybe we can talk thru this next week? I'm not really sure I have all the info at hand here to help make a decision (and it's Friday afternoon and my brain is fried).

Reply:
+1 (albeit from deep on the sidelines) for not making this type of decision on Friday afternoon, particularly one with lovely (distractingly) sunny weather...